### PR TITLE
Doc refinement trsp

### PR DESCRIPTION
**Pull request by cvvergara**
- Refining documentaton
- Refining restrictions data for examples
- Refining combinations data for examples
- Adjusting tests to new table names and data

@pgRouting/admins

### CODE REVIEW

**Review by omshinde — APPROVED** [review]

**Review by krashish8 — COMMENTED** [review]
I think you forgot to update the `NEWS`.

**Comment by cvvergara** [comment]
Indeed, will fix during the review with the gsoc students

File: doc/src/proposed.rst [hunk 1]
```diff
@@ -67,6 +78,12 @@ Proposed Functions
    :start-after: index from here
    :end-before: index to here
 
+:doc:`TRSP-family`
```
Comment (cvvergara) [inline doc/src/proposed.rst:81]: Looks like its been added twice

File: doc/trsp/pgr_trsp.rst [hunk 2]
```diff
@@ -70,29 +70,53 @@ The main characteristics are:
...
+.. rubric:: deprecated
+
+.. code-block:: none
+
    pgr_trsp(sql text, source_edge integer, source_pos float8, target_edge integer, target_pos float8,
-            directed boolean, has_rcost boolean [,restrict_sql text])
+            directed boolean, has_rcost boolean [,restrict_sql text]) -- deprecated on v3.4
    RETURNS SETOF (seq, id1, id2, cost)
 
+.. rubric:: prototype
```
Comment (cvvergara) [inline doc/trsp/pgr_trsp.rst:116]: put prototype first and then deprecated

**Pull request merged**